dev/devicelab/README.md

```diff
@@ -48,28 +54,20 @@ Notably, it will start and stop Gradle, for instance.
 
 ### Running all tests
 
-To run all tests defined in `manifest.yaml`, use option `-a` (`--all`):
-
-```sh
-../../bin/cache/dart-sdk/bin/dart bin/run.dart -a
-```
-
-This defaults to only running tests supported by your host device's platform
-(`--match-host-platform`) and exiting after the first failure (`--exit`).
+`TODO(chillers): Update to use .ci.yaml. https://github.com/flutter/flutter/issues/74660`
 
 ### Running specific tests
 
 To run a test, use option `-t` (`--task`):
 
 ```sh
 # from the .../flutter/dev/devicelab directory
-../../bin/cache/dart-sdk/bin/dart bin/run.dart -t {NAME_OR_PATH_OF_TEST}
+../../bin/cache/dart-sdk/bin/dart bin/test_runner.dart test -t {NAME_OR_PATH_OF_TEST}
```

Comment:
Does `bin/cache/dart-sdk/bin/dart dev/devicelab/bin/test_runner.dart` not work?  Or does `test_runner` need the working directory to be `dev/devicelab`?
I usually ran the `dev/bots/test.dart` variant from the top level 
```sh
SHARD=tool_integration_tests dart dev/bots/test.dart
```

Reply:
Also, can you add directions for how to run a shard?  That's usually what I'm doing locally.

Reply:
Should that information instead be in `dev/bots`? I'm not aware of there being devicelab specific shards other than the integration tests.

Reply:
Oh you're right I was thinking of integration tests, please ignore.